db/fixtures/ae_datastore/ManageIQ/ConfigurationManagement/AnsibleTower/Operations/StateMachines/Job.class/__methods__/launch_ansible_job.rb

```diff
@@ -0,0 +1,91 @@
+#
+# Description: Launch a Ansible Job Template and save the job id
+#              in the state variables so we can use it when we
+#              wait for the job to finish.
+#
+
+class LaunchAnsibleJob
+  ANSIBLE_VAR_REGEX = Regexp.new(/(.*)=(.*$)/)
+  SCRIPT_CLASS = 'ConfigurationScript'.freeze
+  JOB_CLASS = 'ManageIQ_Providers_AnsibleTower_ConfigurationManager_Job'.freeze
+
+  def initialize(handle)
+    @handle = handle
+  end
+
+  def main
+    target
+    job_template
+    run
```

Comment:
This seems odd to me.  I would thing that `target` and `job_template` would return objects that get passed into the `run` method and do not have to be cached in instance variables.

Also, since we fail if job_template is not found that would be the one you want to run first.

Thinking this should be: `run(job_template, target)`